darc/darc.go

```diff
@@ -964,6 +982,9 @@ func (id Identity) GetPublicBytes() []byte {
 		return buf
 	case 4:
 		return id.EvmContract.Address[:]
+	case 6:
+		buf := id.TSM.PublicKey
+		return buf
```

Comment:
Isn't `case 5:` missing?

Reply:
Yes, this is probably missing. I think the DID hasn't been completely implemented...